### DISCUSSION TITLE
Entity graph generation not working with Spring Boot 3 and Hibernate 6

### DISCUSSION THREAD

**Discussion by vicenteqs** [Cosium/spring-data-jpa-entity-graph#150] [Q&A] [answered]
### What steps will reproduce the problem ?
1. Spring boot project with Spring Boot 3.1.3 version
2. I need to use org.hibernate.orm.hibernate-jpamodelgen instead of org.hibernate.hibernate-jpamodelgen because of hibernate 6 in spring boot 3

### What is the expected output ?
XEntityGraph classess generated
### What happens instead ?
No EntityGraph classess generated

### Environment
 - Spring Data JPA version: 3.1.3
 - Hibernate 6.2.7
 - spring-data-jpa-entity-graph version: 3.2.1

**Comment by reda-alaoui** [comment]
You are maybe hit by https://hibernate.atlassian.net/browse/HHH-17362 .
If not, recheck your project configuration, the generator works perfectly on many projects I work on.

**Reply by vicenteqs** [reply, marked as answer]
Thanks for your answer and sorry for late reply.

Yes, it was a bad project configuration, I messed up some parent versions and everything is working fine now. 

Thanks!
